src/styles/index.ts

```diff
@@ -2813,6 +2827,18 @@ const styles = (theme: ThemeColors) =>
             width: 'auto',
         },
 
+        subscriptionCardIcon: {
+            padding: 10,
+            backgroundColor: theme.border,
+            borderRadius: 10,
+        },
+
+        subscriptionEmptyCardIcon: {
```

Comment:
These styles look identical to the styles above. Why are we writing them twice?

Reply:
removed